app/views/users/passwords/new.html.erb

```diff
@@ -0,0 +1,22 @@
+
+<div class="w-screen h-screen flex">
+  <div class="max-w-[600px] max-h-[650px] min-h-[650px] w-full m-auto rounded rounded-lg shadow-[0_0_20px_0_rgba(0,0,0,0.15)] flex py-6">
+
+    <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { class: 'w-full mx-14 flex flex-col gap-y-5' }) do |f| %>
+      <%= render "/shared/errors", resource: resource %>
+      <%= image_tag 'twitter_icon', class:'w-8 h-8 mx-auto mb-0.75rem' %>
+      <h2 class="heading-2 font-bold text-center">Find your twitter account</h2>
+      <p class="description-1">Enter the emai associated with your account to change your password.</p>
+      <div class="field w-full">
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email', class: 'input' %>
+      </div>
+
+      <div class="actions mt-auto mb-0">
```

Comment:
is this actions class doing anything? If not, you can remove it for now. Also I believe the `mb-0` is not necessary. Feel free to leave it there if it's needed